src/less/progress-stepper/progress-stepper.less

```diff
@@ -1,9 +1,19 @@
 @import "../variables/variables.less";
 @import "../mixins/private/token-mixins.less";
 
-@progress-stepper-icon-size: 24px;
 @progress-stepper-min-width: 80px;
 @progress-stepper-large-min-width: 120px;
+// Shortcut private variables
+@_progress-stepper-active-gradient: var(
```

Comment:
Wait, I thought `less` vars couldn't be set using css props.

Reply:
So I think the way it works is that we can't pass this to any "function" to get an output (like fade or something). But we are simply using it as a variable so we don't type the same things over and over.